.github/workflows/fail_on_error.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
When does this script get used? The workflow doesn't seem to run it.

Reply:
@FintasticMan Apologies, it must have not reflected for some reason. The initial motivation was to have a step that calls this script to force the workflow to fail if any errors are found. Is this something of interest to you or shall I remove it altogether?